deployer/src/deployment/run.rs

```diff
@@ -369,15 +379,15 @@ async fn run(
             info!(response = ?response.into_inner(),  "start client response: ");
 
             // Wait for stop reason
-            let reason = stream.message().await.unwrap().unwrap();
+            let reason = stream.message().await.unwrap();
```

Comment:
Should we handle this error, or change it to expect to make it clear why it shouldn't fail?